src/main/java/uk/gov/hmcts/ccd/domain/types/CaseDataValidator.java

```diff
@@ -150,8 +152,11 @@ private List<ValidationResult> validateSimpleField(final String fieldId,
                 .map(result ->
                     new ValidationResult(result.getErrorMessage(), fieldIdPrefix + result.getFieldId()))
                 .collect(Collectors.toList()))
-                .orElseThrow(() ->
-                    new RuntimeException("System error: No validator found for " + fieldType.getType()));
+            .orElseThrow(() -> {
+                log.error(
```

Comment:
Use string formatting with {} instead of concatenating with +

Reply:
Changed error message to use a formatted string.